lib/mongoose_push_web/schemas/request/send_notification/deep/silent_notification.ex

```diff
@@ -1,14 +1,29 @@
 defmodule MongoosePushWeb.Schemas.Request.SendNotification.Deep.SilentNotification do
   require OpenApiSpex
+  alias MongoosePushWeb.Protocols.RequestDecoder
   alias MongoosePushWeb.Schemas.Request.SendNotification.Deep
 
   OpenApiSpex.schema(%{
     title: "Request.SendNotification.Deep.SilentNotification",
     description: "In this request data field is mandatory.",
     type: :object,
-    properties: Deep.base()[:properties],
+    properties: Map.merge(Deep.base()[:properties], Deep.data()[:properties]),
     required: Deep.base()[:required] ++ Deep.data()[:required],
     example: Map.merge(Deep.base()[:example], Deep.data()[:example]),
     additionalProperties: false
   })
+
+  defimpl RequestDecoder,
+    for: MongoosePushWeb.Schemas.Request.SendNotification.Deep.SilentNotification do
+    alias MongoosePushWeb.Protocols.RequestDecoderHelper
+
+    @spec decode(%Deep.SilentNotification{}) :: MongoosePush.request()
+    def decode(schema) do
+      %{
+        service: String.to_atom(schema.service),
```

Comment:
Don't we have a helper that does this better? Even if not, then let's use to_existing_atom at least.

Reply:
same as above